csharp/tests/Valkey.Glide.IntegrationTests/TestConfiguration.cs

```diff
@@ -15,14 +15,17 @@ public class TestConfiguration : IDisposable
     public static List<(string host, ushort port)> STANDALONE_HOSTS { get; internal set; } = [];
     public static List<(string host, ushort port)> CLUSTER_HOSTS { get; internal set; } = [];
     public static Version SERVER_VERSION { get; internal set; } = new();
+    public static bool TLS = false;
```

Comment:
Perhaps this should be a property with an internal set.
I don't get why these properties are public though.

Reply:
Fixed
All of them are public in scope of IT suite.